pkg/report/writer_test.go

```diff
@@ -0,0 +1,41 @@
+package report
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewWriter(t *testing.T) {
+	testCases := []struct {
+		input    string
+		expected string
+	}{
+		{"Hello\tWorld", "Hello:World"},
+		{"Be\tGood", "Be::Good"},
```

Comment:
Not exactly sure how this case works but just checking because it seems off;
Hello World has a single colon : and Be Good has two ::